<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>552</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>552</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>552</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>TeRa client</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#background { background-image: url(&quot;:/images/background.png&quot;); }</string>
  </property>
  <layout class="QVBoxLayout" name="MainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="background">
     <property name="styleSheet">
      <string notr="true">QLabel, QAbstractButton, QAbstractItemView, QComboBox
{ color: #355670; font: 12px &quot;Arial, Liberation Sans&quot;; }

/* widgets */
QPushButton
{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #406A9E, stop: 0.75 #00355F);
border: 0px;
border-radius: 2px;
color: white;
margin-left: 1px;
margin-right: 1px;
min-width: 75px;
padding: 1px 7px;
}

QDialogButtonBox QPushButton, #home QPushButton
{ padding: 4px 7px; }

QPushButton:hover, QPushButton:focus
{
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #F1C157, stop:0.28 #FADF91, stop:1 #EBA927);
color: #00355F;
}

QPushButton:disabled
{ background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #A3AFC0, stop: 0.75 #80909D); }

#viewEmail, #viewBrowse, #viewPrint, #viewEncrypt, #viewFileNameSave, #viewSaveAs, #signAddFile
{
color: #509B00;
text-align: left;
text-decoration: underline;
background: transparent;
padding: 0px;
margin: 0px;
min-width: 0px;
}

#viewSaveAs { text-align: right; }

/* header */
#headerLine, #headerLine2, #headerLine3
{ background-color: #00355F; }

#cards, #languages
{
background-color: transparent;
border: 0px;
margin-right: 0px;
color: white;
}

#cards QListView, #languages QListView
{
background-color: white;
border: 0px;
color: #355670;
}

#cards::down-arrow, #languages::down-arrow
{ image: url(&quot;:/images/languages_button.png&quot;); }

#cards::drop-down, #languages::drop-down
{ border: 0px; }

#settings, #help, #about
{
background-color: transparent;
color: #afc6d1;
font: bold;
min-width: 0px;
}

#infoFrame
{
background-color: rgba( 255, 255, 255, 153 );
border-radius: 3px;
color: #668696;
padding-left: 10px;
padding-right: 10px;
}
#infoFrame QLabel, #infoFrame QLineEdit, #infoFrame QCheckBox
{
color: #668696;
}

#infoFrame QLabel, #signInfo 
{ font: bold; }

/* content */
#warning {
color: #668696;
background-color: rgba(255,255,255,200);
border: 1px solid #cddbeb;
border-radius: 5px;
padding: 10px;
}

#homeButtonLabel,
#viewContentLabel, #viewSignaturesLabel,
#signContentLabel, #signSigner, #signSignerLabel
{
color: #00355F;
font: bold;
}

#home, #intro, #sign, #view,
#viewSignatures
{ background-color: transparent; }

#signContentFrame, #viewContentFrame
{
background-color: rgba( 255, 255, 255, 200 );
border: 1px solid gray;
color: #355670;
}

#signContentView, #viewContentView
{
background-color: transparent;
padding: 5px;
}

#viewSignatures
{ color: #71889A; }

#signSignerRole, #viewSignaturesScroll
{
background-color: rgba( 255, 255, 255, 200 );
border: 1px solid gray;
padding: 5px;
}

#version
{ color: #668696; }</string>
     </property>
     <layout class="QVBoxLayout" name="backgroundLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="headerLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <spacer name="headerSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>45</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="settings">
          <property name="text">
           <string>Settings</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="Line" name="headerLine">
          <property name="maximumSize">
           <size>
            <width>2</width>
            <height>20</height>
           </size>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="help">
          <property name="text">
           <string>Help</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="Line" name="headerLine2">
          <property name="maximumSize">
           <size>
            <width>2</width>
            <height>20</height>
           </size>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="about">
          <property name="text">
           <string>About</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="Line" name="headerLine3">
          <property name="maximumSize">
           <size>
            <width>2</width>
            <height>20</height>
           </size>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ComboBox" name="languages">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>20</height>
           </size>
          </property>
          <property name="accessibleName">
           <string>Languages</string>
          </property>
          <item>
           <property name="text">
            <string notr="true">Estonian</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">English</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">Russian</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="infoLayout">
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>115</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="infoSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnStamp">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Find &amp;&amp; Timestamp</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelProcess">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="logText">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 30);</string>
        </property>
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="version">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="pixmap">
            <pixmap resource="../../common/images/common_images.qrc">:/images/EL_Regionaalareng_horisontaal.png</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>ComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../common/images/common_images.qrc"/>
  <include location="../../../common/images/common_images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonGroup</sender>
   <signal>buttonClicked(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonClicked(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>parseLink(QString)</slot>
  <slot>buttonClicked(int)</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
